pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceReader.java

```diff
@@ -25,13 +25,13 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.ObjectWriter;
 import com.google.common.util.concurrent.RateLimiter;
+
```

Comment:
This change is unnecessary, we'd better keep the PR clear.

Reply:
Thanks for your suggestion! I have deleted these changes.